src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,68 @@
+.stopwatch {
+  position: relative;
+  width: $width;
+  height: $width;
```

Comment:
The `$width` variable is used for both width and height, which could be confusing if the stopwatch is not meant to be a square. Make sure that the design requires a square shape, otherwise consider using a more appropriately named variable, for example `$size`.